lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -465,15 +467,27 @@ class Type
       TASK_TRIGGER_SESSION_STATE_CHANGE  = 11
     end
 
-    TYPE_MANIFEST_MAP = {
-      Type::TASK_TRIGGER_DAILY => 'daily',
-      Type::TASK_TRIGGER_WEEKLY => 'weekly',
+    TIME_BASED_TRIGGER_MAP = {
```

Comment:
This should probably be `SCHEDULE` instead of `TIME` ... but we can change later - NBD